src/stochprog.jl

```diff
@@ -0,0 +1,112 @@
+using LightGraphs
+
+export AbstractStochasticProgram, stochasticprogram
+export isleaf, probability, cutgenerator, numberofpaths
+export setprobability!, add_scenario_state!, add_scenario_transition!
+
+"""
+    AbstractStochasticProgram
+
+Stochastic program instance
+"""
+abstract type AbstractStochasticProgram <: AbstractGraph{Int} end
+
+"""
+    stochasticprogram(args...)
+
+Creates a stochastic program from the arguments
+"""
+function stochasticprogram end
```

Comment:
StochasticProgram? CamelCase?

Reply:
I am open to using CamelCase. I am used to using lowercase in such cases since it is a function and not a type.